src/playwright/playwright.config.ts

```diff
@@ -16,33 +16,43 @@ export default defineConfig({
   outputDir: path.join(process.cwd(), 'dist', 'playwright'),
   projects: [
     {
-      name: 'aria',
-      snapshotPathTemplate: './src/aria-snapshots/{arg}{ext}',
-      testMatch: /.*\.test\.aria\.ts/,
+      name: 'functionality',
```

Comment:
Let me know what you think. But the plan is to put every other non-visual, non-lint rule test into this project. So all of these too:

- `*.test.basics.ts`
- `*.test.events.ts`
- `*.test.focus.ts`
- `*.test.forms.ts`
- `*.test.interactions.ts`